src/main/java/net/kodehawa/mantarobot/commands/currency/item/Items.java

```diff
@@ -188,6 +188,7 @@
             // ---------------------------------- 5.4 PET ITEMS START HERE ----------------------------------
             INCUBATOR_EGG = new Item(ItemType.PET, "\uD83E\uDD5A", "Incubator Egg", "items.incubator_egg", "items.description.incubator_egg", 300, false, false, "4;3;1", 11, 12, 18),
             WATER_BOTTLE = new Item(ItemType.PET, "", "Water Bottle", "items.water_bottle", "items.description.water_bottle", 100, true, true),
+            MAGIC_WATCH = new Item(ItemType.COLLECTABLE, "⏳", "Magical Watch", "items.magic_watch", "items.description.magic_watch",0, false, false)
```

Comment:
shouldn't this be the unicode representation of the emote for consitency sake?

Reply:
My bad. I use eclipse instead of IntelliJ

Reply:
That shouldn't matter though, I usually just look up the unicode (java) representation of it on google or something :P